scripts/legislation/africa/lesotho_english.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+
+function remove_all_text_before_first_header {
+  sed -n '/^\Enacted by the Parliament of Lesotho\./,$p' | \
+    sed -n '/^PART I/,$p' 
+}
+
+function prefix_article_numbers_with_article_literal {
+  sed -E 's/^([0-9]+\.)/Article \1/'
+}
+
+function replace_parentheses_around_article_delimiters_with_square_brackets {
+  sed -E 's/^\(([A-Za-z0-9]+)\)/[\1]/'
+}
+
+function move_article_titles_after_article_numbers {
+  sed -En '${p;q};N;/\nArticle/{s/^(.*)\n(Article [0-9]+\.) /\2 \1|/p;b};P;D'
```

Comment:
Please remind me what the lowercase `b` does in this command. The rest of it looks great.

Reply:
I am not entirely sure. This piece of code is sourced from the [namibia_english.sh](https://github.com/danpdraper/eia/pull/57) where you condensed my code into what is above. 

You referenced [this](https://stackoverflow.com/a/4513283) Stack Overflow answer.

Reply:
Oh, I see. Okay, thanks.